regression/goto-instrument/replace-calls-05/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--replace-calls f:g --replace-calls h:f
+Function f cannot both be replaced and be a replacement
```

Comment:
What happens if I replace the entry point (potentially specified by --function)?

Reply:
Can you add a test involving function pointers?

Reply:
Ok, I've added a test that involves function pointers.

Regarding the entry point replacement with `--function`, were you meaning that the error message should be more specific to indicate that it's coming from the replace calls analysis?

Reply:
If it is saying the usual 'No entry point found' then this is fine. An invariant violation would not be ok.

Reply:
The replace calls pass works both when an entry point is present and when not (`goto-instrument` does not check whether an entry point is present). Also it cannot be used to replace the entry point, as it replaces calls not functions and the entry point is not called by anything.